### PR TITLE
Added .highlight command

### LINKED ISSUE #1039: How do you clear highlights when debugging with REPL?
### Expected behavior
I should be able to clear highlights after using highlight() without starting over.

### Actual behavior
All elements highlighted with highlight() sticks until I refresh the page.

### Steps to reproduce
1. taiko
2. goto()
3. highlight()

### Versions
```
node 13.6.0
taiko 1.0.3
```
Some webapps when refreshing brings you to a completely different screen.  It would be nice if clear() or something similar could remove the highlights.

### PR DESCRIPTION
**Pull request by BugDiver**
Added `.highlight` with following options

* .highlight disable
* .highlight enable
* .highlight clear

Fixes #1039

### CODE REVIEW

File: lib/taiko.js [hunk 1]
```diff
@@ -1621,19 +1621,58 @@ module.exports.highlight = highlight;
...
+  let style = document.createElement('style');
+  let css = '.taiko_highlight_style { outline: 0.5em solid red; }';
+  head.appendChild(style);
+  style.type = 'text/css';
+  style.id = taikoHighlightStyleID;
+  style.appendChild(document.createTextNode(css));
+}
+
+/**
+ * Clear all highlights for the current page.
```
Comment (sriv) [inline lib/taiko.js]: change to:

`Clear all highlights marked using @highlight on the current page.`

File: package.json [hunk 2]
```diff
@@ -52,14 +52,15 @@
...
+    "documentation": "^12.1.3",
     "extract-zip": "^1.6.6",
     "fs-extra": "^9.0.0",
     "https-proxy-agent": "^5.0.0",
     "is-reachable": "^4.0.0",
     "progress": "^2.0.3",
     "proxy-from-env": "^1.0.0",
     "recast": "^0.18.5",
-    "documentation": "^12.1.3"
+    "typescript": "^3.8.3"
```
Comment (sriv) [inline package.json]: why is there a `typescript` dependency?

File: lib/taiko.js [hunk 3]
```diff
@@ -1654,7 +1654,7 @@ function addTaikoHighlightStyleIfNotPresent() {
 }
 
 /**
- * Clear all highlights for the current page.
+ * Clear all highlights marked using @highlight on the current page.
```
Comment (sriv) [inline lib/taiko.js]: Sorry - the syntax needs to use `@link` to link `highlight` properly. See https://jsdoc.app/tags-inline-link.html for example.

Currently this looks like this: https://deploy-preview-1146--ecstatic-raman-953525.netlify.com/#clearhighlights

File: lib/taiko.js [hunk 4]
```diff
@@ -1621,19 +1621,58 @@ module.exports.highlight = highlight;
 
 async function highlight(selector, ...args) {
   validate();
+  if (!defaultConfig.highlightOnAction) {
+    throw new Error('Highlights are disabled. Please enable highlights.');
```
Comment (zabil) [inline lib/taiko.js]: This should be a warning

Comment (BugDiver) [reply lib/taiko.js]: Done

**Pull request merged**